src/Accounts/Accounts/Rest/InvokeAzRestMethodCommand.cs

```diff
@@ -111,91 +125,309 @@ public override void ExecuteCmdlet()
                 this.Path = Uri.PathAndQuery;
             }
 
-            IAzureRestClient serviceClient = null;
-            if (ByPath.Equals(this.ParameterSetName) || ByParameters.Equals(this.ParameterSetName))
+            IAzureRestClient serviceClient = this.InitializeServiceClient();
+
+            AzureOperationResponse<string> response = ExecuteRestRequest(serviceClient);
+            WriteObject(new PSHttpResponse(response));
```

Comment:
(High priority) I prefer not writing the first response when it's a LRO cause it can be confusing to customers why one request can have two responses.

Reply:
Resolved this issue.